.github/updatecli/values.d/scm.yml

```diff
@@ -1,5 +1,8 @@
 scm:
   enabled: true
   owner: elastic
-  repository: docs-builder
+  repositories: |
+    docs-builder
+    edot-cloud-forwarder-aws
+    edot-cloud-forwarder-azure
```

Comment:
This is not yet available; let me request IT.

BTW, we will eventually move away from our GH app and use a new approach for managing ephemeral tokens, see https://github.com/elastic/ci-gh-actions/pull/3

Reply:
I don't see the [edot-cloud-forwarder-gcp repo](https://github.com/elastic/edot-cloud-forwarder-gcp)